### PR TITLE
add eslint config to repo to integrate with text editors

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.1 to e90bceca9f7b81b3d0c2d0554d776a4366a936af
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.1 to 4d6f7b31ff410002a4d4ecd95d01fecdd11ef774
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.1 to b271957f71861865c71f3146ee9402ce7561a57b
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.1 to 94a9003a828b50848de8a57d767951e332998f97
No significant bundle size differences found.

File: demos/src/imperative.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const lastMonth = new Date();
 lastMonth.setMonth(now.getMonth() - 6);
 
 times[0].setAttribute('datetime', today.toISOString());
-times[1].setAttribute('datetime', new Date(today.getTime() - (1000 * 60 * 60 * 20)).toISOString());
+times[1].setAttribute('datetime', new Date(today.getTime() - 1000 * 60 * 60 * 20).toISOString());
```
Comment (notlee) [inline demos/src/imperative.js]: nice 👍

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.1 to cb37cd8b09f627f84499e0436e15cbf35a52791d
No significant bundle size differences found.

**Pull request merged**